Add FreshWorks jobs to README.md

jared-freshworks opened the pull request.

jaddison commented:
Hey Jared, can you take a look at the failed build "Details" page and clean up the 404'd links in your PR? Thanks!

jared-freshworks commented:
> Hey Jared @jared-freshworks, can you take a look at the failed build "Details" page and clean up the 404'd links in your PR? Thanks!



> Hey Jared @jared-freshworks, can you take a look at the failed build "Details" page and clean up the 404'd links in your PR? Thanks!



> Hey Jared @jared-freshworks, can you take a look at the failed build "Details" page and clean up the 404'd links in your PR? Thanks!

Hi jaddison. I have fixed what was failing. There were two dead links so I removed them. Thanks!
